fedbiomed/common/cli.py

```diff
@@ -135,6 +137,47 @@ def initialize_create_configuration(self) -> None:
                  "If the configuration file exists, leave it unchanged"
         )
 
+        create.add_argument(
```

Comment:
New syntax for configuration creation is not backward compatible (`fedbiomed_run node configuration create` and `fedbiomed_run researcher configuration create` don't work anymore). So we need:

1. either to provide backward compatible syntax
2. or update all the references existing. Found some in `fedbiomed_run` + in documentation and tutorials + in notebooks + in VPN configuration scripts ...

Reply:
Thank you Sergen, I see now all mentions have updated syntax